displayio_cartesian.py

```diff
@@ -42,36 +54,36 @@
 
 
 class Cartesian(Widget):
-    """A cartesian widget.  The origin is set using ``x`` and ``y``.
+    """A cartesian widget.  The origin is set using :const:`x` and :const:`y`.
```

Comment:
I might be misunderstanding the use of `const` here. Doesn't `const` imply that these values cannot be changed?

But it is possible for these to change isn't it? Changing x and y allows user to move the cartesian to a different location on the screen.

Reply:
Right, right, I will change it to attribute.